integration-cli/docker_cli_search_test.go

```diff
@@ -11,83 +12,50 @@ func (s *DockerSuite) TestSearchOnCentralRegistry(c *check.C) {
 	testRequires(c, Network, DaemonIsLinux)
 
 	out, exitCode := dockerCmd(c, "search", "busybox")
-	if exitCode != 0 {
-		c.Fatalf("failed to search on the central registry: %s", out)
-	}
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
```

Comment:
Ah didn't saw that.. You can actually ignore `exitCode` (`out, _ := exitCode(…)`) as it always will be equal to `0` if `dockerCmd` didn't failed. There is some PR to remove that (to be reopened though :sweat_smile:).